tools-v2/pkg/cli/command/curvebs/status/copyset/copyset.go

```diff
@@ -0,0 +1,106 @@
+/*
+*  Copyright (c) 2023 NetEase Inc.
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+ */
+/*
+* Project: CurveCli
+* Created Date: 2023-05-18
+* Author: chengyi01
+ */
+
+package copyset
+
+import (
+	"fmt"
+	"strings"
+
+	cmderror "github.com/opencurve/curve/tools-v2/internal/error"
+	cobrautil "github.com/opencurve/curve/tools-v2/internal/utils"
+	basecmd "github.com/opencurve/curve/tools-v2/pkg/cli/command"
+	checkcopyset "github.com/opencurve/curve/tools-v2/pkg/cli/command/curvebs/check/copyset"
+	clustercopyset "github.com/opencurve/curve/tools-v2/pkg/cli/command/curvebs/list/copyset"
+	"github.com/opencurve/curve/tools-v2/pkg/config"
+	"github.com/opencurve/curve/tools-v2/pkg/output"
+	"github.com/spf13/cobra"
+)
+
+type CopysetCommand struct {
+	basecmd.FinalCurveCmd
+}
+
+var _ basecmd.FinalCurveCmdFunc = (*CopysetCommand)(nil) // check interface
+
+func NewCopysetCommand() *cobra.Command {
+	return NewStatusCopysetCommand().Cmd
+}
+
+func (cCmd *CopysetCommand) AddFlags() {
+	config.AddRpcTimeoutFlag(cCmd.Cmd)
+	config.AddRpcRetryTimesFlag(cCmd.Cmd)
+	config.AddBsMdsFlagOption(cCmd.Cmd)
+	config.AddBsMarginOptionFlag(cCmd.Cmd)
+}
+
+func (cCmd *CopysetCommand) Init(cmd *cobra.Command, args []string) error {
+	config.AddBsFilterOptionFlag(cCmd.Cmd)
```

Comment:
add flag in `AddFlags`?

Reply:
> add flag in `AddFlags`?

This is for the called command, not a parameter of the command.